fdbrpc/FlowTransport.h

```diff
@@ -238,4 +253,36 @@ inline bool Endpoint::isLocal() const {
 	return addresses.address == localAddrs.address || (localAddrs.secondaryAddress.present() && addresses.address == localAddrs.secondaryAddress.get());
 }
 
+struct IChecksum {
```

Comment:
Is there any reason not to move IChecksum and CRC32 to FlowTransport.actor.cpp? It doesn't really feel like it belongs in the public interface of the FlowTransport module.